test/blackbox-tests/test-cases/pkg/outdated.t

```diff
@@ -140,14 +143,14 @@ With multiple contexts, the errors should also be printed for each context.
   - dune.workspace.lock is up to date.
   - dune.lock is up to date.
   Error: Some packages could not be found.
-  - When checking dune.workspace.lock, the following packages:
-    - bar
-    - foo
-    were not found in the following opam repositories:
-    - None
-  - When checking dune.lock, the following packages:
-    - bar
-    - foo
-    were not found in the following opam repositories:
-    - None
+  When checking dune.workspace.lock, the following packages:
```

Comment:
The enumeration here is used for things that don't really belong in a list. I got rid of it.

Reply:
My intention with that was to make it easier to read multiple error messages, but I guess this can be improved in other ways.